Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.yml

```diff
@@ -390,6 +390,15 @@ script:
       name: filter
       required: false
       secret: false
+    - default: false
+      description: Retrieves the total count of matching resources.
+      isArray: false
+      name: count
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'true'
```

Comment:
Shouldn't this be true and false? as options?

Reply:
I could only find information about using the argument with 'true'
Also, when I tried to run it with 'false' I encountered an exception.